nextbike/cli.py

```diff
@@ -56,9 +56,36 @@ def visualize():
 def predict():
     print("Read in trips file...")
     df_trips = io.read_trips()
-    df_trips.drop("Unnamed: 0", axis=1, inplace=True)
-    X_train, X_test, y_train, y_test = prediction.simple_split(df_trips)
-    #prediction.train(X_train, y_train)
+    df_trips.drop(["Unnamed: 0", "Place_start", "Place_end"], axis=1, inplace=True)
+    print("Create dummie variables...")
+    df = prediction.create_dummies(df_trips)
+    print("Cast datetime...")
+    df_fin = prediction.cast_datetime(df)
+    print("Analyse correlations...")
+    prediction.corr_analysis(df)
+    # ___________________________________________________________________________________
+    print("Split data in train and test set...")
+    X_train, X_test, y_train, y_test = prediction.simple_split(df_fin)
+    print("Scale data...")
+    X_scaled_train = prediction.scale(X_train)
+    print("Do PCA...")
+    X_train_transformed = prediction.do_pca(X_scaled_train)
+    # ____________________________________________________________________________________
+    print("Train regression...")
+    prediction.train_linear_regression(X_train_transformed, y_train)
+    print("Predict with regression")
+    prediction.predict_by_regression(X_test, y_test)
+
+
+def test(df):
+    time = 0
+    a = df[df["End Time"] <= time]
+    # - 0 places
+    b=a.sort_by(by=["place_id", "datetime_end"])
+    c = b.drop_duplicates(["place_id"], keep="last")
+    print(c["p_bikes_end"])
```

Comment:
Funktion umbennen - und soll die hier in der cli.py bleiben?

Reply:
Just a reminder...
Prototype function for visualize in one moment of time all stations... 
Has to be implemented in visualization.

Reply:
Done